### PR TITLE
[STAKAN-70] Add estimated APY

### PR DESCRIPTION
**Pull request by AndrewKhizhnyak**
[Task](https://ankrnetwork.atlassian.net/browse/STAKAN-70)

[Design](https://www.figma.com/file/mQ9yM5xfy57NASyuT8ZG0z/Stkr-Transform-ANKR?node-id=1277%3A128)

Screenshot 2021-07-05 at 16 02 00

### CODE REVIEW

File: src/store/createStore.ts [hunk 1]
```diff
@@ -28,9 +30,16 @@ export const createApplicationStore = ({
...
-    driver: createDriver({
-      processResponse: response => ({ data: response }),
-    }),
+    driver: {
+      default: createDriver({
+        processResponse: response => ({ data: response }),
+      }),
+      axios: createAxiosDriver(
+        axios.create({
+          baseURL: process.env.REACT_APP_API_BASE,
```
Comment (BigBadAlien) [inline src/store/createStore.ts:39]: Is it dynamic or fixed?

**Comment by BigBadAlien** [comment]
Plz, follow branch naming rules https://ankrnetwork.atlassian.net/wiki/spaces/STAKAN/pages/1666646162/Branches+and+pull+request+naming

File: src/store/actions/AvalancheActions.ts [hunk 2]
```diff
@@ -475,4 +475,19 @@ export const AvalancheActions = {
...
     }),
   ),
+  fetchEstimatedAPY: createAction(
+    'FETCH_ESTIMATED_APY',
+    (): RequestAction => ({
+      request: {
+        promise: (async () => {
+          const stkrSdk = StkrSdk.getForEnv();
+          const { apy } = await stkrSdk.getEstimatedAPY();
+          return parseFloat(apy).toFixed(2) + '%';
```
Comment (Kud8) [inline src/store/actions/AvalancheActions.ts:485]: it's better to use ``${parseFloat(apy).toFixed(2)} %``

Comment (AndrewKhizhnyak) [reply src/store/actions/AvalancheActions.ts:485]: why?

File: src/modules/api/gateway.ts [hunk 3]
```diff
@@ -158,6 +158,11 @@ export interface INotarizeTransferRequest {
   transactionHash: string;
 }
 
+export interface IEstimatedAPYReply {
+  apy: string;
```
Comment (BigBadAlien) [inline src/modules/api/gateway.ts:162]: apy: BigNumber

File: src/store/actions/AvalancheActions.ts (hunk 2 above)
Comment (Kud8) [reply src/store/actions/AvalancheActions.ts:485]: it's easer to change and add some prefix or suffix

Comment (Kud8) [reply src/store/actions/AvalancheActions.ts:485]: and for me it's more readable

**Pull request closed without merging**